src/World.cpp

```diff
@@ -1199,7 +1199,7 @@ void cWorld::TickMobs(std::chrono::milliseconds a_Dt)
 				else
 				{
 					auto & Wolf = static_cast<cWolf &>(Monster);
-					if (Wolf.IsAngry())
+					if (!Wolf.IsAngry())
 					{
 						Monster.Destroy(true);
```

Comment:
Will this destroy tamed wolves as well?

Reply:
It shouldn't, wolves only get angry when they aren't tamed https://github.com/cuberite/cuberite/blob/master/src/Mobs/Wolf.cpp#L58

Reply:
But this is destroying wolves that are _not_ angry.